app/controllers/users_controller.rb

```diff
@@ -1,5 +1,18 @@
 class UsersController < ApplicationController
-  def show; end
+  def show
+    conn = Faraday.new(url: 'https://api.github.com') do |faraday|
+      faraday.headers['Content-Type'] = 'application/json'
+      faraday.headers['Authorization'] = "token #{current_user.token}"
+    end
+    response = conn.get('/user/repos')
+    @repos = JSON.parse(response.body, symbolize_names: true)
+
+    follower_list = conn.get('/user/followers')
+    @followers = JSON.parse(follower_list.body, symbolize_names: true)
+
+    following_list = conn.get('user/following')
+    @following = JSON.parse(following_list.body, symbolize_names: true)
+  end
```

Comment:
Good first pass at red / green / refactor. We'll want to refactor all of these methods out into abstract objects with single responsibilities.